nuxt-frontend/components/backend-development/LandingSection.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <section class="bg-ios-landing-background bg-opacity-10 3xl:outer-container">
+    <div
+      class="container flex flex-col lg:flex-row pt-6 lg:py-[7.5rem] gap-4 text-center lg:text-left"
+    >
+      <div class="flex flex-col gap-4 lg:gap-6 lg:flex-1">
+        <h1 class="header-1 text-black-core/[0.87] lg:w-1/2">
+          Backend development company
+        </h1>
+        <p class="sub-h1-regular xs:px-4 lg:!px-0 text-black-core/[0.6]">
+          Our backend development team isn’t just about writing code; it's about
+          crafting powerful, efficient, scalable, and secure foundations that
+          drive innovative applications. We shape, optimize, and elevate digital
+          platforms ensuring they're ready for tomorrow's challenges.
+        </p>
+        <nuxt-link
+          class="gradient-btn primary-btn ml-0 hidden lg:flex"
+          to="/contact"
+          @click.native="$mixpanel.track('tap_backend_landing_cta')"
+        >
+          <span class="sub-h3-semibold xl:sub-h4-semibold"
+            >Be Future-Ready With Us</span
+          ></nuxt-link
+        >
+      </div>
+      <div class="flex flex-col -gap-1 lg:flex-1 lg:my-auto">
+        <div class="w-4/5 md:w-3/5 lg:w-[35rem] mx-auto">
```

Comment:
why we need to use this type of width here without it its also looking good

Reply:
Without this the image would fit the size of the screen and it will exceed the screen and the button will not be visible

Reply:
i am talking from lg devices not mobile devices